### PR TITLE
Add uk.me.doof.Lander

### PR DESCRIPTION
**Pull request by nickg**
This is a Lunar Lander style arcade game using GL/SDL2

### CODE REVIEW

**Comment by nickg** [comment]
bot, build uk.me.doof.Lander

**Comment by nickg** [comment]
bot, build uk.me.doof.Lander

File: uk.me.doof.Lander.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "uk.me.doof.Lander",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
```
Comment (LeandroStanger) [inline uk.me.doof.Lander.json]: Update runtime to 19.08

**Comment by nickg** [comment]
bot, build uk.me.doof.Lander

File: uk.me.doof.Lander.json (hunk 1 above)
Comment (nickg) [reply uk.me.doof.Lander.json]: Done, thanks.

File: uk.me.doof.Lander.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/nickg/lander/releases/download/r0.7.1/lander-0.7.1.tar.xz",
+                    "sha256": "1ddeb57039597eb89ca1b2c1b4c5acd2a0b2b44bfc09ee419042fd836fc937e2"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```
Comment (bilelmoussaoui) [inline uk.me.doof.Lander.json]: Nitpick: finish-args should be just before modules.

**Comment by nickg** [comment]
bot, build uk.me.doof.Lander

**Comment by bilelmoussaoui** [comment]
One last thing, do you own the uk.me.doof domain name?

File: uk.me.doof.Lander.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "uk.me.doof.Lander",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "lander",
+    "finish-args": [
+        "--socket=x11",
```
Comment (nedrichards) [inline uk.me.doof.Lander.json]: you need --share=ipc as well if you have x11

**Comment by nickg** [comment]
bot, build uk.me.doof.Lander

File: uk.me.doof.Lander.json (hunk 3 above)
Comment (nickg) [reply uk.me.doof.Lander.json]: Done, thanks.

**Comment by nickg** [comment]
> One last thing, do you own the uk.me.doof domain name?

Yes.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/uk.me.doof.Lander

**Pull request closed without merging**